docs/CUSTOMIZING.md

```diff
@@ -0,0 +1,66 @@
+# Customizing guide
+
+This document describes how you can create your own app for an event.
+
+## 1. Required information
+
+The following information are needed to configure the app for your event.
+
+- Schedule URL which provides Frab compatible XML
+- Session URL template, e.g. `https://awesome-event.com/2021/events/%1$s.html`
+- Server backend type, one of: `pentabarf`, `frab`, `pretalx`
+- Schedule feedback URL template (optional), e.g. `https://awesome-event.com/2021/events/%s/feedback/new`
+- Google Play URL, e.g. `https://play.google.com/store/apps/details?id=com.awesome.event.schedule`
+- Event URL, e.g. `https://awesome-event.com/2021`
+- Start and end date of the event
+- Email address for bug reports
+- Engelsystem URL (optional), e.g. `https://engelsystem.de/awesome-event/shifts-json-export?key=YOUR_KEY`
+
+## 2. Required graphic assets and colors
+
+The following graphics and colors are needed in order to customize the look and feel of the app.
```

Comment:
At least for me (not a regular app creator) it would help if you would include a screenshot that points out the different icons on a real phone screen with an already installed app. Because I dont know in what places / situations these icons will be used.

Reply:
Nice idea. :heavy_check_mark: Added in f49d549452dbe6b2e10b8050ace391d0ee12b55f.